src/mixins/canvas_events.mixin.js

```diff
@@ -368,6 +453,10 @@
         return;
       }
 
+      if (!this._isMainEvent(e)) {
+        return;
+      }
+
```

Comment:
this repeated for down/up/move is needed to avoid doing stuff in fabricjs object transformations when we are outside the single pointer movement, main touch changes.